src/pages/dashboard/staking-pools/StakingPoolsRow.tsx

```diff
@@ -63,67 +67,87 @@ const StakingPoolsRow: React.FC<
 
   const { balances } = stakingBalancesData;
 
-  // const safetyPoolStakingBalances = balances[StakingPool.Safety];
-  // const safetyPoolData = stakingPoolsData.data[StakingPool.Safety];
-
-  // let safetyPoolSize: React.ReactNode;
-
-  // if (safetyPoolData.poolSize) {
-  //   const { num, suffix } = abbreviateNumber({
-  //     num: MustBigNumber(safetyPoolData.poolSize).toString(),
-  //     decimals: DecimalPlaces.None,
-  //   });
-
-  //   safetyPoolSize = (
-  //     <ValueWithIcon>
-  //       <NumberFormat thousandSeparator displayType="text" value={num} suffix={suffix} />
-  //       <AssetIcon size={AssetIconSize.Small} symbol={AssetSymbol.DYDX} />
-  //     </ValueWithIcon>
-  //   );
-  // } else {
-  //   safetyPoolSize = defaultLoadingBar;
-  // }
-
-  // let safetyPoolUserBalance: React.ReactNode = '-';
-
-  // if (walletAddress) {
-  //   if (safetyPoolStakingBalances.userBalance) {
-  //     const { num, suffix } = abbreviateNumber({
-  //       num: MustBigNumber(safetyPoolStakingBalances.userBalance).toString(),
-  //       decimals: DecimalPlaces.Abbreviated,
-  //     });
-
-  //     safetyPoolUserBalance = (
-  //       <ValueWithIcon>
-  //         <NumberFormat thousandSeparator displayType="text" value={num} suffix={suffix} />
-  //         <AssetIcon size={AssetIconSize.Small} symbol={AssetSymbol.DYDX} />
-  //       </ValueWithIcon>
-  //     );
-  //   } else {
-  //     safetyPoolUserBalance = defaultLoadingBar;
-  //   }
-  // }
+  const safetyPoolStakingBalances = balances[StakingPool.Safety];
+  const safetyPoolData = stakingPoolsData.data[StakingPool.Safety];
+
+  let formattedSafetyPoolSize: React.ReactNode;
+
+  const {
+    poolSize: safetyPoolSize,
+    rewardsPerSecond: safetyRewardsPerSecond,
+  } = stakingPoolsData.data[StakingPool.Safety];
```

Comment:
`safetyPoolData` is already extracted from `stakingPoolsData` above

Reply:
💯